src/Sentry/Protocol/Envelopes/Envelope.cs

```diff
@@ -320,6 +321,18 @@ public static Envelope FromTransaction(Transaction transaction)
         return new Envelope(eventId, header, items);
     }
 
+    /// <summary>
+    /// Creates an envelope that contains one or more Metrics
+    /// </summary>
+    internal static Envelope FromMetrics(IEnumerable<Metric> metrics)
+    {
+        var header = DefaultHeader;
+
+        var items = metrics.Select(EnvelopeItem.FromMetric).ToArray();
```

Comment:
bonus point to remove this LINQ method call with a loop to avoid an allocations 😅